lib/intelligent_foods.rb

```diff
@@ -42,10 +42,15 @@ def base_url
     end
 
     def client
-      @client =
+      @client ||=
         IntelligentFoods::ApiClient.new(id: client_id, secret: client_secret)
     end
 
+    def refresh_client
+      @client = nil
+      client
```

Comment:
this breaks command/query separation. the code should ask for the client to be refreshed and then ask for the client